fs/march/march.go

```diff
@@ -416,17 +417,29 @@ func (m *March) processJob(job listDirJob) ([]listDirJob, error) {
 	}
 
 	// If NoTraverse is set, then try to find a matching object
-	// for each item in the srcList
+	// for each item in the srcList to head dst object
+	ci := fs.GetConfig(m.Ctx)
+	limiter := make(chan bool, ci.Checkers)
```

Comment:
`bool` is 4 data bytes, `struct{}` is 0 data bytes
let's save a little ram

Reply:
done